ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -320,18 +322,6 @@ open class BottomCommandingController: UIViewController {
         updateSheetHeaderSizingParameters()
     }
 
-    public override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
```

Comment:
No, we should never be dismissing VCs we don't own. This dismissal exists specifically to avoid a case where popovers turn into modal system sheets when horizontal size categories change. We never want to show the popover in a system sheet, so we dismiss it just to be safe.

Reply:
Another reason this hook is bad is that it was firing on iPhone, and this is a problem specific to iPad. But the dismiss bug is breaking iPhone clients.